tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -20,6 +23,37 @@ def __init__(self):
         super(GlusterIntegrationSdsSyncStateThread, self).__init__()
         self._complete = gevent.event.Event()
 
+    def _emit_event(self, resource, curr_value, msg):
+        alert = {}
+        alert['source'] = 'tendrl-gluster-integration'
+        alert['pid'] = os.getpid()
+        alert['timestamp'] = tendrl_now().isoformat()
+        alert['alert_type'] = 'status'
+        severity = "info"
+        if curr_value == "Stopped":
+            severity = "critical"
+        alert['severity'] = severity
+        alert['resource'] = resource
```

Comment:
The alerting module does need to know about the object definition and attributes. Tendrl will be generating alerts on specific attributes of a Tendrl object.

But if you are expecting a underscore between the entity/object and alert/attribute we can continue with below scheme

<object>_<attribute>

works?